julia/README.md

```diff
@@ -0,0 +1,60 @@
+# MXNet
+
+[![MXNet](http://pkg.julialang.org/badges/MXNet_0.6.svg)](http://pkg.julialang.org/?pkg=MXNet)
+
+
+MXNet.jl is the [dmlc/mxnet](https://github.com/dmlc/mxnet) [Julia](http://julialang.org/) package. MXNet.jl brings flexible and efficient GPU computing and state-of-art deep learning to Julia. Some highlight of its features include:
```

Comment:
Please update the dmlc/mxnet URL to [MXNet](https://github.com/apache/incubator-mxnet)

Reply:
highlights